docs/Installation.md

```diff
@@ -33,3 +33,48 @@ If `python` is not yet installed on your machine, you should add it.
 Error: connect EACCES /var/run/docker.sock
 ```
 Follow [these](https://docs.docker.com/install/linux/linux-postinstall/) instructions to create `docker` group and add your user.
+
+# Example Ubuntu Install
+This script automates steps 1-5 above and includes troubleshooting fixes. It was created for Ubuntu 18. 
+```
+#update environment
+sudo apt-get update && sudo apt-get -y upgrade
+sudo apt-get --assume-yes install \
+   apt-transport-https \
+   ca-certificates \
+   curl \
+   gnupg-agent \
+   software-properties-common 
+#docker install steps
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+sudo apt-key fingerprint 0EBFCD88
+sudo add-apt-repository \
+  "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) \
+  stable"
+sudo apt-get update
+sudo apt-get --assume-yes install docker-ce docker-ce-cli containerd.io
+sudo docker run hello-world
```

Comment:
I wouldn't think this is needed. Can you argue why you are including this as part of the script?

Reply:
There wasn't much consideration on what's needed vs. not needed. for example, on second review the hello-world command is just a waste of time. But yea, this is just a distillation of the install links provided.